barretenberg/cpp/src/barretenberg/ecc/scalar_multiplication/scalar_multiplication.cpp

```diff
@@ -857,21 +857,26 @@ typename Curve::Element evaluate_pippenger_rounds(pippenger_runtime_state<Curve>
 
 template <typename Curve>
 typename Curve::Element pippenger_internal(typename Curve::AffineElement* points,
-                                           typename Curve::ScalarField* scalars,
+                                           std::span<const typename Curve::ScalarField> scalars,
                                            const size_t num_initial_points,
                                            pippenger_runtime_state<Curve>& state,
                                            bool handle_edge_cases)
 {
+    size_t num_initial_points_power_2 = 1 << numeric::get_msb(num_initial_points);
```

Comment:
Can you please add a sente commenting these three lines

Reply:
These shouldn't be here actually right now, will remove